xml/operations-maintenance-upgrade_maintenance.xml

```diff
@@ -0,0 +1,1835 @@
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <literal>nova-console-auth</literal>
+            </para>
+          </listitem>
+        </itemizedlist>
+        <note>
+          <para>
+            This should resolve the errors that cause the <filename>config-processor-run.yml</filename>
+            playbook to fail.
+          </para>
+        </note>
+      </step>
+      <step>
+        <para>
+          Similarly, remove any service components that are redundant and no
+          longer required. This means that you should comment out (or delete)
+          any lines for the following service components:
+        </para>
+        <itemizedlist>
+          <listitem>
+            <para>
+              <literal>ceilometer-api</literal>
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <literal>glance-registry</literal>
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <literal>manila-share</literal>
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <literal>neuton-lbaasv2-agent</literal>
+            </para>
+          </listitem>
+        </itemizedlist>
+        <note>
+          <para>
+            This should resolve most of the warnings reported by the
+            <filename>config-processor-run.yml</filename> playbook.
+          </para>
+        </note>
+        <important>
+          <para>
+            If you have deployed the <systemitem class="service">designate</systemitem> service components
+            (<systemitem class="service">designate-pool-manager</systemitem> and <systemitem class="service">designate-zone-manager</systemitem>) in your cloud,
+            you will see warnings like those shown above, indicating that these
+            service components have been deprecated.
+          </para>
+          <para>
+            You can switch to using the newer <systemitem class="service">designate-worker</systemitem> and <systemitem class="service">designate-producer</systemitem>
+            service components, which will quieten these deprecation warnings
+            produced by the <filename>config-processor-run.yml</filename> playbook run.
+          </para>
+          <para>
+            However, this is a procedure that should be perfomed after the
+            upgrade has completed, as outlined in the <xref linkend="post-upgrade-tasks"/>
+            section below.
+          </para>
+        </important>
+     </step>
+     <step>
+      <para>
+        Once you have made the necessary changes to your input model, if
+        you run <command>git diff</command> under the <filename>~/ardana/openstack/my_cloud/definition</filename>
+        directory, you should see output similar to the following:
+      </para>
+      <screen>&prompt.ardana;cd ~/openstack/my_cloud/definition
```

Comment:
Once again, the paths do not match:
`~/ardana/openstack/my_cloud`
vs
`~/openstack/mycloud`
@rtamalin ...?